CHANGELOG.md

```diff
@@ -26,6 +26,12 @@
 <!--          addition of notable features. When PATCH is 0 it can      -->
 <!--          be omitted                                                -->
 
+# 13 April 2021
+# ATT&CK website version 3.2.0
+### Improvements
+- STIX2.1 support.
+- STIX notes support for object pages.
+
```

Comment:
- Should mention that data can now be pulled from any http or filesystem endpoint instead of using hardcoded data paths (#139).

Reply:
I'll make the change on the branch since it's so small.